tests/www/prescribers_views/test_list.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+from django.urls import reverse
+from pytest_django.asserts import assertContains
+
+from itou.prescribers.enums import PrescriberOrganizationKind
+from tests.prescribers.factories import (
+    PrescriberOrganizationFactory,
+    PrescriberOrganizationWithMembershipFactory,
+)
+
+
+pytestmark = pytest.mark.ignore_template_errors
+
+
+def test_list_accredited_organizations(client):
+    organization = PrescriberOrganizationWithMembershipFactory(
+        authorized=True,
+        kind=PrescriberOrganizationKind.DEPT,
+        membership__is_admin=True,
+    )
+    user = organization.members.filter(prescribermembership__is_admin=True).get()
+    client.force_login(user)
```

Comment:
Aussi simple, une requête en moins.
```suggestion
    organization = PrescriberOrganizationFactory(authorized=True, kind=PrescriberOrganizationKind.DEPT)
    membership = PrescriberMembershipFactory(is_admin=True, organization=organization)
    client.force_login(membership.user)
```